src/pocketmine/network/mcpe/protocol/StructureBlockUpdatePacket.php

```diff
@@ -26,16 +26,32 @@
 #include <rules/DataPacket.h>
 
 use pocketmine\network\mcpe\NetworkSession;
+use pocketmine\network\mcpe\protocol\types\StructureEditorData;
 
-class StructureBlockUpdatePacket extends DataPacket{
+class StructureBlockUpdatePacket extends DataPacket/* implements ServerboundPacket*/{
```

Comment:
This should not be here

Reply:
I believe that it has a place to be.
It is a "little help" for merging into the master branch.

Reply:
It's been added to all the new stuff since that interface was added on the master branch. It saves having to dig into the binary twice when we implement a bunch of new stuff.